edxval/models.py

```diff
@@ -404,6 +404,7 @@ class VideoTranscript(TimeStampedModel):
     """
     Transcript for a video
     """
+    video = models.ForeignKey(Video, related_name='video_transcripts')
     video_id = models.CharField(max_length=255, help_text='It can be an edx_video_id or an external video id')
```

Comment:
@nasthagiri I am unable to have foreign key named `video` due to this previous `video_id`, its clashing with `video` :(
```bash
edxval.VideoTranscript.video_id: (models.E006) The field 'video_id' clashes with the field 'video' from model 'edxval.videotranscript'.
```
One solution I could think of is rename `video` to `related_video` if that sounds alright? OR Is there any better way for this?

Reply:
Wouldn't you want to delete the `video_id` column now - since it's redundant with `video`?

Reply:
yes but I don't want to generate a migration for that.

Reply:
in the same release.

Reply:
Why not?  Are these tables already populated in production?

Reply:
Not yet. but according to https://openedx.atlassian.net/wiki/spaces/ENG/pages/162953307/Migration+Don+ts, we should have two releases if we are dropping a column.

Reply:
Oh I think the old code will still find the `video_id`? (which will be the foreign key) Right?

Reply:
Yes, that is true.
But, will there be "old code" running in production even that would be accessing this table?
That wiki applies to the case where you have both "old" and "new" code running in prod during blue/green deployments.

Reply:
Old code and the new code are running for the same DB, so yes, old code will be accessing this table.

Reply:
But the good thing is no one's using this model as of today.

Reply:
So, I want it to be finalized first.

Reply:
> But the good thing is no one's using this model as of today. 
Exactly.  Since no LMS requests will access this model, you can go ahead and make the changes in a single migration without needing to worry about blue-green deployments.

By "old code", we mean code running the previous version of the platform that needs to run when HTTP requests are handled by the previous version (in a blue-green deployment).